db/migrations/.eslintrc

```diff
@@ -0,0 +1,3 @@
+{
+  extends: "lob/migrations", 
+}
```

Comment:
Could you make sure there are newlines at the end of these files (Github shows this missing newline character when they're missing)? Looks like this file and the `.travis.yml` are missing them. It's just a POSIX standard that all files should end in newlines, so you'll want to always follow that.

Reply:
done